scripts/pickle_wikidata.py

```diff
@@ -28,4 +36,4 @@
 
 
 with open("H:\\Courses\\RA\\wikidata-water-features\\scripts\\hydro_matcher_new.pkl", "rb") as file:
-    vocab, matcher, wikidata = pickle.load(file)
\ No newline at end of file
+    vocab, matcher, wikidata = pickle.load(file)
```

Comment:
Ok no need to update this file. In order to update the waterwheel vocab, do the following:

1. Create csv files (with proper names) and place them in `scripts/data` folder.
2. Run the `build_vocab_csvs` function from the `scripts/util.py` file.
3. This will create a file: `scripts/data/doc_bins.msgpack` with the updated vocabulary.
4. Move this file over to `waterwheel/resources`.
5. Test `WaterWheel` class with this new file.